src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/subscription/SubscriptionController.java

```diff
@@ -131,7 +131,7 @@ public ResponseEntity<Subscription> findBySubId(@Parameter @PathVariable UUID su
     @Operation(summary = "Returns the list of relevant subscriptions associated with a given user id.")
     @GetMapping("/user/{userId}")
     public ResponseEntity<UserSubscription> findByUserId(@Parameter @PathVariable String userId) {
```

Comment:
Can the Path Variable be changed to UUID instead of String here? Would save us needing to do the UUID cast directly

Reply:
fixed.